packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -75,9 +75,11 @@ export interface DashboardLayoutProps {
    * @default null
    */
   branding?: Branding | null;
+  // @TODO: Remove deprecated prop from v0.16.0 onwards
   /**
    * Navigation definition for the dashboard. [Find out more](https://mui.com/toolpad/core/react-dashboard-layout/#navigation).
    * @default []
+   * @deprecated Set the navigation in the [AppProvider](https://mui.com/toolpad/core/react-app-provider) instead.
    */
   navigation?: Navigation;
```

Comment:
Probably also want to show a warning at runtime.

Reply:
added in https://github.com/mui/toolpad/pull/4881/commits/9551fd1d36a117bb5e03dbe5ea016452775068c3